clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2421,7 +2421,7 @@ device kernel, the attribute is not ignored and it is propagated to the kernel.
 If the`` intel::reqd_work_group_size`` or ``cl::reqd_work_group_size``
 attribute is specified on a declaration along with a
 intel::num_simd_work_items attribute, the work group size attribute
-argument (the first argument) must be evenly divisible by the argument specified
+argument (the last argument) must be evenly divisible by the argument specified
```

Comment:
This documentation also needs to be updated for the OpenCL vs SYCL confusion. This would be a great place for a big warning to users because they're definitely going to be surprised by this SYCL design choice.

Reply:
Done